### PR TITLE
Feature/member identities c 1026

### PR DESCRIPTION
**Pull request by themarolt**
# Changes proposed ✍️
- Move member identities to a dedicated table `memberIdentities` from their current place in `members.username` column.
- Add `username` column to `activities` to see which particular member identity added the activity

## Checklist ✅
- [x] Label appropriately with `Feature`, `Enhancement`, or `Bug`.
- [x] Tests are passing.
- [x] New backend functionality has been unit-tested.
- [ ] ~Environment variables have been updated:~
  - [ ] ~Local frontend configuration: `frontend/.env.dist.local`, `frontend/.env.dist.composed`.~
  - [ ] ~Local backend: `backend/.env.dist.local`, `backend/.env.dist.composed`.~
  - [ ] ~[Configuration docs](https://docs.crowd.dev/docs/configuration) have been updated.~
  - [ ] ~Team members only: update environment variables in override, staging and production env. files and trigger update config script.~
- [ ] ~API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).~
- [x] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.
- [x] All changes have been tested in a staging site.
- [x] All changes are working locally running crowd.dev's Docker local environment.

### CODE REVIEW

**Review by epipav — COMMENTED** [review]
I'm testing review webhooks, please ignore :-)

**Review by epipav — COMMENTED** [review]
test 2

**Review by epipav — APPROVED** [review]
lgtm :+1:  included one nitpick

File: backend/src/database/repositories/memberRepository.ts [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
 
 class MemberRepository {
   static async create(data, options: IRepositoryOptions, doPopulateRelations = true) {
+    if (!data.username) {
+      throw new Error('Username not set when creating member!')
+    }
+
+    const platforms = Object.keys(data.username) as PlatformType[]
+    if (platforms.length === 0) {
+      throw new Error('Username not set when creating member!')
```
[the remark this answers is not in the document]
Comment (themarolt) [reply backend/src/database/repositories/memberRepository.ts]: Done

[the remark this answers is not in the document]
Comment (epipav) [reply backend/src/database/repositories/memberRepository.ts]: b

**Pull request merged**